client/src/utils/userInfo.js

```diff
@@ -1,8 +1,10 @@
 export const getInitials = (firstName, lastName) => {
   return firstName.charAt(0).toUpperCase() + lastName.charAt(0).toUpperCase();
 };
-
 export const getInitialsFromFullName = (fullName) => {
   const [firstName, lastName] = fullName.split(" ");
   return firstName.charAt(0).toUpperCase() + lastName.charAt(0).toUpperCase();
 };
+export const getOrgInitials = (firstName) => {
+  return firstName.charAt(0).toUpperCase()
+};
```

Comment:
You probably want to do something similar to lines 4-6. Where the full org name is passed into the function returns only the first of each word that was separated by space. Here you can also only return two initials, of the very first two words on the org name, so sort of split the array, and return the first char of the first word and the first char of the second word, ignore everything else for now.